chrome/content/zotero-better-bibtex/schomd.coffee

```diff
@@ -117,8 +191,10 @@ Zotero.BetterBibTeX.schomd.citation = (citekeys, {style, libraryID} = {}) ->
 
   return '' if itemIDs.length == 0
 
-  url = "http://www.zotero.org/styles/#{style ? 'apa'}"
-  style = Zotero.Styles.get(url)
+  urlstyle = "#{style ? 'apa'}"
+  style = Zotero.Styles.get("http://www.zotero.org/styles/#{urlstyle}")
+  style = Zotero.Styles.get("http://juris-m.github.io/styles/#{urlstyle}") unless style != false
```

Comment:
Is there a particular non-`false` but false-ish value (`null`, `undefined`, `""` or `0`) you expect here? If not, simply `style ||= ` would have sufficed instead of `unless style != false`